scripts/g.extension/g.extension.py

```diff
@@ -1273,6 +1273,63 @@ def install_extension_xml(edict):
     return None
 
 
+def filter_multi_addon_addons(mlist):
+    """Filter out list of multi-addon addons which contains
+    and installs only *.html manual page, without source/binary
+    excutable module and doesn't need to check metadata.
+
+    e.g. i.sentinel multi-addon install i.sentinel addon which
+    install only i.sentinel.html file.
+
+    :param list mlist: list of individual multi-addon addons
+
+    :return list mlist: list of individual multi-addon addons
+                        without addons which install only *.html
+                        manual page
```

Comment:
```suggestion
    :param list mlist: list of multi-addons (groups of addons
                        with respective addon overview HTML pages)

    :return list mlist: list of individual multi-addons without respective
                        addon overview HTML pages
```

... also a suggestion.

Reply:
I agree with the suggestions.